### ISSUE TITLE
--exclude option

### ISSUE THREAD

**Issue by mhradilek** [psss/did#107]
When some systems are down one would still like to create status report based on the whole config just excluding the systems that are not relevant.

I propose the `--exclude` option where one would define systems to be omitted even though they are present in configuration.

Example:
`did last week --exclude bz,jira,wiki`

**Comment by optak** [comment]
Good idea,
I would just extend it with another option to disable all section with given type, for example
`did --exclide-type git,github`

**Comment by kkaarreell** [comment]
I was just going to file the very same request. I just wanted to propose rather options with --no-* prefix. Why? Currently there is an option like --bz covering all --bz-* specific options. Similarly, having --no-bz option could work as an exclude and IMHO it would fit better into the user interface than --exclude.

**Comment by psss** [comment]
Agree, with @kkaarreell: Using `--no-section` seems to be a little bit more consistent. Should be relatively easy to implement.
